src/views/contact.hbs

```diff
@@ -30,24 +30,29 @@
       </p>
       <div class="w-100 mb-4" id="mapid"></div>
     </div>
-    <form class="col-sm-4 mb-4">
+    <form action="/contact" id="contact-form" method="post" role="form" class="col-sm-4 mb-4">
       <h2>{{{__ "contact_subtitle"}}}</h2>
       <div class="form-group">
         <div class="row">
           <div class="col">
-            <input type="text" class="form-control form-control-sm" id="name" placeholder="{{{__ "common_name"}}}"
-              aria-label="{{{__ "common_name"}}}" required>
+            <input type="text" class="form-control form-control-sm" id="name" name="name"
+              placeholder="{{{__ "common_name"}}}" aria-label="{{{__ "common_name"}}}" required>
           </div>
           <div class="col">
-            <input type="email" class="form-control form-control-sm" id="email" aria-describedby="emailHelp"
-              placeholder="{{{__ "common_email"}}}" required>
+            <input type="email" class="form-control form-control-sm" id="email" name="email"
+              aria-describedby="emailHelp" placeholder="{{{__ "common_email"}}}" required>
           </div>
         </div>
       </div>
       <div class="form-group">
-        <textarea type="text" class="form-control form-control-sm" id="message" rows="10"
+        <textarea type="text" class="form-control form-control-sm" id="message" name="message" rows="10"
           placeholder="{{{__ "common_letter"}}}" aria-label="{{{__ "common_letter"}}}" required></textarea>
       </div>
+      <div class="form-group">
+        <label class="control-label" for="verification">{{{__ "contact_verify"}}}</label>
+        <input type="text" class="form-control form-control-sm" id="verification" name="verification"
+          placeholder="{{{__ "contact_verification"}}}">
+      </div>
```

Comment:
Why do we need to verify?

Reply:
I just saw it in an example and thought it doesn't hurt to prevent spam. You are right though, it might confuse some people and I guess the chance for someone setting up a bot to spam Lernwerkstatt is rather low 😄 Let me know if I should remove it.